changes/1001.misc.rst

```diff
@@ -0,0 +1 @@
+The packaging of sdist and wheel artefacts was simplified.
```

Comment:
```suggestion
The packaging of sdist and wheel artifacts was simplified.
```

Reply:
Artefact is also a valid spelling, and is the preferred spelling (in my style guide, anyway); it's from the Latin "arte factum".

Reply:
Ah, I didn't know that English has it in this form.